docs/developer-guide/working-with-custom-formatters.md

```diff
@@ -99,7 +102,7 @@ module.exports = function ( results ) {
     }, { errors: 0, warnings: 0 } );
 
     if ( summary.errors > 0 || summary.warnings > 0 ) {
-        console.log( 'Errors: ' + summary.errors + ', Warnings: ' + summary.warnings + '\n' );
+        return 'Errors: ' + summary.errors + ', Warnings: ' + summary.warnings + '\n';
     }
```

Comment:
I think there should be a `return '';` statement here so that the formatter always returns a string.

Reply:
Thanks for the feedback! I have amended it according to your suggestion.